internal/benchrunner/runners/rally/runner.go

```diff
@@ -266,12 +267,29 @@ func (r *runner) setUp() error {
 	}
 	r.scenario = scenario
 
-	err = r.installPackage()
-	if err != nil {
-		return fmt.Errorf("error installing package: %w", err)
+	if len(r.options.PackageFromRegistry) > 0 {
```

Comment:
Nit. We could keep the `installPackage` method, and add this new logic there.

Reply:
it would be a very long method that it would be better to split in two different methods anyway, imho :)

Reply:
yes! that's what I did in the end :D